dmutils/content_loader.py

```diff
@@ -4,49 +4,49 @@
 import os
 
 
-class ContentLoader(object):
-
-    def __init__(self, manifest, content_directory):
-
-        section_order = self._read_yaml_file(manifest)
+class ContentBuilder(object):
 
+    def __init__(self, manifest, content_directory, yaml_loader):
```

Comment:
Since all the file loading is part of `YAMLLoader` content builder doesn't really need the `manifest` and `content_directory`. I think we could move the `.get_question` method to `YAMLLoader` and change this to:
```python
yaml_loader = YAMLLoader(manifest, content_directory)
content = ContentBuilder(yaml_loader)
```